src/app/components/more-fewer/more-fewer.tsx

```diff
@@ -41,7 +49,7 @@ export function Paginated({children, perPage=10}) {
     }
     useEffect(() => {
         if (pageChanged) {
-            $.scrollTo(ref.current, 70);
+            $.scrollTo(ref.current as HTMLDivElement, 70);
```

Comment:
Don't you want to check for current being set before calling this?  I checked the helper and it doesn't check for a null param so it seems like this could cause issues...

Reply:
As I understand it, when the `useEffect` runs, the ref will be set, since it's set in the return value for the component. If I do a test for not-set, I won't be able to cover that branch with testing.

Reply:
I added an `assertNotNull` helper function.